docs/how-tos/how_to_skip_rmas_in_returns.mdx

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 2
+---
+
+# How to skip RMAs in returns
+
+We'll see how you can disable RMAs as a prerequisite to creating a customer
+return. Solidus' [built-in process for returns][returns] is mighty and
+flexible. Only that sometimes a store doesn't need such a feature-rich system.
+
+Go ahead and create an order from the storefront. You must go through
+everything, from adding items to the cart to confirming the payment.
+
+Next, you need to capture the payments in the system and perform the shipments.
+You can do it whether from the admin panel or through the console. E.g.:
+
+```ruby
+order = Spree::Order.find_by_number('R723438584')
+order.payments.map(&:capture!)
+order.shipments.map(&:ship!)
+```
+
+Let us go to the heart of the matter. When you add return items to a customer's
+return, Solidus checks whether they are eligible for that. By default, it uses
```

Comment:
I find that, in English, being a bit more explicit often comes across as more natural than using implicit references (e.g., "for this", "for that"):
```suggestion
return, Solidus checks whether they are eligible for being returned. By default, it uses
```

Reply:
Thanks. Integrated.